galvasr2/align/spark/schemas.py

```diff
@@ -45,7 +45,7 @@
             StructField("aspect_ratio", StringType(), True),
             StructField("backup_location", StringType(), True),
             StructField("closed_captioning", StringType(), True),
-            StructField("collection", ArrayType(StringType(), True), True),
+            StructField("collection", StringType(), True),
```

Comment:
I thought you had already made this change in this past PR: https://github.com/mlcommons/peoples-speech/pull/8/files#diff-49bd79597919909effba2db47b8ad03e3f7a3645f28ae9113571e0a5e700dd79R48

Are you sure that `juanciro/order-code` is rebased on top of the main branch? Contact me privately if you're not sure what rebasing means.